README.md

```diff
@@ -166,6 +166,10 @@ Returns a new `FindWrapper` with a subset of the previously selected elements gi
 
 Uses the same possible selectors as [`RenderContext#find(selector)`](#rendercontextfindselector).
 
+### `FindWrapper#name()`
+Returns the name of node.
+This can only be called on a wrapper of a single node.
```

Comment:
Can you think of a small usage/test assertion to include here?

Reply:
Done :)